src/main/java/uk/gov/hmcts/dts/fact/model/Facility.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.dts.fact.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class Facility {
+    private String description;
+    private String name;
+
+    public Facility(uk.gov.hmcts.dts.fact.entity.Facility facility) {
```

Comment:
.

Reply:
starting to notice this is a thing, might be wrong so ill ignore the other ones.